### ISSUE TITLE
Hey! Your "Displace" Plugin for FF14 is outdated. Could you please update it? <3

### ISSUE THREAD

**Issue by Kumicoo** [Drakansoul/SoulsMsg#1]
Hey. As the title says, the Displace plugin for FF14 is outdated. Sadly, I do not know how to contact you any other way than to make an Issue on another project of yours. I am really sorry for that (if there is another way, please do tell?).

We would really appreciate it if you could update Displace. <3 Thank you

**Comment by Drakansoul** [comment]
Yeah, I'll get around to it at some point. Dont have dalamud beta access because i can't be bothered to get it and have a lot going on right now but when those things get sorted i can likely update it to work again.

**Comment by Kumicoo** [comment]
How to enable Dalamud Beta:
Open a file %appdata%\XIVLauncher\dalamudConfig.json with any text editor (windows notepad will do)
find and set parameter "DalamudBetaKind" to "stg"
find and set parameter "DalamudBetaKey" to "burrito"
Save the file and restart the game.


Not writing this to make you update faster, just letting you know. Dalamud beta access is literally just a password they update in their public github repo, in case you dont wanna wait before going back into ff14!

I hope your things get sorted soon. Hope it's not too bad and if they are that they get resolved. <3
